Cargo.toml

```diff
@@ -1,3 +1,23 @@
 [workspace]
 
 members = ["acir_field", "acir", "acvm", "stdlib"]
+
+[workspace.package]
+version = "0.4.1"
```

Comment:
This breaks some work I'm doing so we can automated releases for each package.

Reply:
No worries, I assumed that we were keeping these in sync (as they currently are)